openedx/frontend-app-ora-grading
# fix: enable scroll in grading modal

## Description
This pull request introduces scrollbars to the MFE to improve the user experience when dealing with large content areas.

## Testing instructions
1. Deploy this to an instance.
1. Submit an ORA with many images and/or a large amount of text.
1. Go to the Instructor Dashboard > Open Assessment and start grading.
1. Ensure scrollbars show in the fullscreen modal window.

## References
- https://github.com/open-craft/frontend-app-ora-grading/pull/5
